solutions/project/hub-env/org-policies/exceptions/compute-disable-serial-port-access-except-hub-project.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#########
+#
+# GCP Organization Policies
+# Org policies that correspond with a Guardrail will contain a label indicating what Guardrails it helps in enforcing
+# https://cloud.google.com/resource-manager/docs/organization-policy/org-policy-constraints
+#
+# Constraint: constraints/compute.disableSerialPortAccess
+#
+# This boolean constraint disables serial port access to
+# Compute Engine VMs belonging to the organization, project,
+# or folder where this constraint is set to True.
+#
+# This exception is for the fortigates VM in the hub project.
+# Serial console access is critical for troubleshooting the appliance.
+#
+#########
+apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
+kind: ResourceManagerPolicy
+metadata:
+  name: compute-disable-serial-port-access-except-hub-project
+  namespace: policies
+  annotations:
+    config.kubernetes.io/depends-on: resourcemanager.cnrm.cloud.google.com/namespaces/projects/Project/hub-project-id # kpt-set: resourcemanager.cnrm.cloud.google.com/namespaces/projects/Project/${hub-project-id}
+  labels:
+    guardrail: "true"
+    guardrail-enforced: guardrail-02
+spec:
+  constraint: "constraints/compute.disableSerialPortAccess"
+  booleanPolicy:
+    enforced: false
```

Comment:
Just wondering if always on is the best policy? Could we not submit a PR to change this policy when an incident occurs?

Reply:
We set it to auto-set enforce a year or so ago as there was the potential to use the serial port for unintended purposes. See https://github.com/GoogleCloudPlatform/pbmm-on-gcp-onboarding/pull/77 for more details. It looked like it was a low risk issue but decided to air on the side of caution.

Reply:
@lucstjean-ssc @cartyc @amcmullin @CraigBaltzer IMO, we keep this "project" level exception. Then, we create a new feature request that will allow enabling the fortigate's serial port (compute.instances.setmetadata) only by a specific group. In order to further mitigate risk, that group should leverage just-in-time access.

Reply:
just an fyi, the serial console is a very important troubleshooting tools for whomever manages fortigate appliances. It is particularly essential when performing firmware upgrades or to troubleshoot any cluster/HA issue.